File: video/src/main/java/com/zhuliyi/video/mvp/model/bean/VideoBean.java
package com.zhuliyi.video.mvp.model.bean;

/**
 * Describe : 视频item
 * Author : zhuly
 * Date : 2019-05-15
 */
public class VideoBean {
    private String id;
    private String title;//标题
    private String videourl;//视频url
    private String imgurl;//封面图片
    private String playcount;//播放次数
    private String totalTimes;//时长
    private String width;//视频宽度
    private String height;//视频高度

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getVideourl() {
        return videourl;
    }

    public void setVideourl(String videourl) {
        this.videourl = videourl;
    }

    public String getImgurl() {
        return imgurl;
    }

    public void setImgurl(String imgurl) {
        this.imgurl = imgurl;
    }

    public String getPlaycount() {
        return playcount;
    }

    public void setPlaycount(String playcount) {
        this.playcount = playcount;
    }

    public String getTotalTimes() {
        return totalTimes;
    }

    public void setTotalTimes(String totalTimes) {
        this.totalTimes = totalTimes;
    }

    public String getWidth() {
        return width;
    }

    public void setWidth(String width) {
        this.width = width;
    }

    public String getHeight() {
        return height;
    }

    public void setHeight(String height) {
        this.height = height;
    }
}

File: video/src/main/java/com/leory/badminton/video/mvp/model/bean/VideoBean.kt
package com.leory.badminton.video.mvp.model.bean

/**
 * Describe : 视频item
 * Author : zhuly
 * Date : 2019-05-15
 */
data class VideoBean(var id: String? = null,
                     var title: String? = null,//标题
                     var videourl: String? = null,//视频url
                     var imgurl: String? = null,//封面图片
                     var playcount: String? = null,//播放次数
                     var totalTimes: String? = null,//时长
                     var width: String? = null,//视频宽度
                     var height: String? = null//视频高度
)